latex/generated-20251113/0001-rfc-process/0001-rfc-process.md

```
@@ -0,0 +1,204 @@
+- Multiple references SHALL be separated by commas
+- If no RFC references exist, the field MUST contain "none"
+- Example:
+  `[RFC-0002](../RFC-0002-mixnet-keywords/0002-mixnet-keywords.md), [RFC-0004](../RFC-0004-hopr-packet-protocol/0004-hopr-packet-protocol.md)`
+
+##### 4.4.2.2. External References
+
+- External references MUST be listed in a dedicated `## References` section at the end of the document
+- References MUST use sequential numbering with zero-padding: [01], [02], etc.
+- In-text citations MUST use the numbered format: "as described in [01]"
+- References SHOULD be formatted in accordance with the following style, based on APA reference style with numeric labels in square brackets:
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)